otaclient/app/downloader.py

```diff
@@ -337,12 +346,11 @@ def _download_task(
         #       the number will be of the decompressed file)
         _hash_inst, downloaded_file_size = self._hash_func(), 0
         # NOTE: traffic_on_wire is the number of bytes we directly downloaded from remote
-        traffic_on_wire = 0
-        _err_count = 0
+        traffic_on_wire, err_count = 0, 0
```

Comment:
Sometimes you change variable name with `_` or w/o `_`.
Please revert this change if you don't have rule.
If there is no explicit rule, this variable name will be modified in the next PR.